src/mumble/GlobalShortcut_win.cpp

```diff
@@ -424,6 +424,39 @@ void GlobalShortcutWin::run() {
 	} while (!isInterruptionRequested());
 }
 
+bool GlobalShortcutWin::isRateLimited() {
+	if (throttleThreshold > 0) {
+		throttleCounter++;
+
+		if (throttleCounter < throttleThreshold) {
```

Comment:
If `throttleThreshold == 0`, this will always block and given we increase the counter before the comparison, the same is true for `throttleThreshold == 1` 🤔 
So maybe this should use `<=` instead of `<`?